core/node/libp2p/rcmgr.go

```diff
@@ -598,3 +602,41 @@ func NetResetLimit(mgr network.ResourceManager, repo repo.Repo, scope string) (r
 
 	return result, nil
 }
+
+func ensureConnMgrMakeSenseVsRessourcesMgr(rcm rcmgr.LimitConfig, cmgr config.ConnMgr) error {
```

Comment:
ensureConnMgrMakeSenseVsRessourcesMgr → ensureConnMgrMakeSenseVsResourceMgr
(extra s' in Ressources)